packages/mobile/ios/Podfile.lock

```diff
@@ -1202,4 +1202,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: f1e56ba6cee31ca0cf79cc470496cc0906977d8f
 
-COCOAPODS: 1.12.0
```

Comment:
Prob shouldn't change?

Reply:
I think this is the latest version? Perhaps everyone should upgrade? installing cocoapods gives you this on our current version of xcode/ruby

Reply:
https://github.com/CocoaPods/CocoaPods/releases

Reply:
Yeah could be?

Reply:
Can just send it and we can all update

Reply:
```
sudo gem install cocoapods
```
https://guides.cocoapods.org/using/getting-started.html#updating-cocoapods

Reply:
I can post out a note